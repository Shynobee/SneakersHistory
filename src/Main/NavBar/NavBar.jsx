import React from 'react';
import PropTypes from 'prop-types';
import { link } from 'react-router';
import {
  withStyles,
  AppBar,
  Toolbar,
  Button
} from '@material-ui/core';

const styles = withStyles({
  root: {
    width: '100vh'
  },
});

class NavBar extends React.Component {

  render() {
    const { classes } = this.props;
    return (

      <div className={classes.root}>
        <AppBar position="static" color="default">
          <Toolbar>
            <Button>
                Sneakers History
            </Button>
            <Button>
                Nike
            </Button>
            <Button>
                Jordan
            </Button>
            <Button>
                Adidas
            </Button>
            <Button>
                Saucony
            </Button>
            <Button>
                Diadora
            </Button>
          </Toolbar>
        </AppBar>
      </div>
    );
  }
}

NavBar.propTypes = {
  classes: PropTypes.object.isRequired,
};

const NavBarWithStyle = withStyles(styles)(NavBar);

export default NavBarWithStyle;