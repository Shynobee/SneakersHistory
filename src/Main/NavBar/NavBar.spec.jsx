import React from 'react';
import ReactDOM from 'react-dom';
import NavBar from './NavBar';

describe('NavBar', () => {
  it('Should renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NavBar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('Should match snapshot', () => {
    expect(NavBar).toMatchSnapshot();
  });
});
