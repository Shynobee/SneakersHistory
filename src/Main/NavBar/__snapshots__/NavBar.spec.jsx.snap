// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Should match snapshot 1`] = `[Function]`;
